### PR TITLE
fix #72616 voe.sx

### LINKED ISSUE #72616: voe.sx
### Issue URL (Annoyance)
 [https://voe.sx/e/vzfdaknmqsg0](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fvoe.sx%2Fe%2Fvzfdaknmqsg0)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (180), <br/>Block Push API, <br/>Block Location API
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS
Custom filter subscriptions: | https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances.txt,<br>https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/privacy.txt

### PR DESCRIPTION
**Pull request by stanislav-atr**
#72616

### CODE REVIEW

**Review by wout-codes — COMMENTED** [review]
@stanislav-atr What do you think of these rules? The rules you created do not work anymore, you can check yourself.
`||highrevenuecpm.com^$document,popup` ==> `EnglishFilter/sections/adservers.txt`
`voe.*,voe.sx#%#//scriptlet("prevent-window-open")` ==> `EnglishFilter/sections/general_extensions.txt`

**Comment by stanislav-atr** [comment]
Yeah, it seems they've changed popup. 

(disabling js required):
<details><summary>Screenshot:</summary>

image
</details><br/>

Since this popup is laready blocked by desktop AG, `$$` is not necessary, i guess.
@AdguardTeam/filters-maintainers, please, take a look.

**Comment by wout-codes** [comment]
> Yeah, it seems they've changed popup.
> 
> (disabling js required):
> 
> Screenshot:
> 
> Since this popup is laready blocked by desktop AG, `$$` is not necessary, i guess.
> @AdguardTeam/filters-maintainers, please, take a look.

Which code?

**Comment by wout-codes** [comment]
@stanislav-atr Which rule do you mean?

File: EnglishFilter/sections/general_extensions.txt
Comment (Alex-302) [inline EnglishFilter/sections/general_extensions.txt:240]: why `*`?

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:240]: There are more domains I am assuming.

Comment (AdamWr) [reply EnglishFilter/sections/general_extensions.txt:240]: It seems that it breaks download button.
Download button is also broken by the rule which I removed already - https://github.com/AdguardTeam/AdguardFilters/commit/b6a8913b8ffbc8788094d40223b70a51d90fa5ff

This rule seems to works fine:
```
voe.sx#%#//scriptlet('abort-current-inline-script', 'EventTarget.prototype.addEventListener', '/location[\s\S]*?getElementsByTagName/')
```
at least until they will change something again.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:240]: They change their code every day

Comment (AdamWr) [reply EnglishFilter/sections/general_extensions.txt:240]: You can enable AdGuard Popup Blocker, it blocks these popups.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:240]: `voe.*#%#//scriptlet("prevent-window-open")` This one does all the job

Comment (AdamWr) [reply EnglishFilter/sections/general_extensions.txt:240]: `voe.*#%#//scriptlet("prevent-window-open") This one does all the job`
It breaks download button.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:240]: hmm..sensitive too

Comment (AdamWr) [reply EnglishFilter/sections/general_extensions.txt:240]: >AdGuard Pop-up Blocker is quite useless. In most of the sites, it fails only some sites it blocks. No offense.

Okay, but it works fine in this case, so I don't see any reason to not use it.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:240]: This site it does the job so yeah

**Review by AdamWr — APPROVED** [review]

**Pull request merged**
